### PR TITLE
Improve ignore documentation.

### PR DESCRIPTION
**Pull request by gjb2048**
Following on from https://github.com/moodlehq/moodle-plugin-ci/issues/104 and https://github.com/moodlehq/moodle-plugin-ci/issues/107 - I've improved the documentation and fixed the example in IgnoringFiles.md as it was a duplicate of the global example.

### CODE REVIEW

File: docs/IgnoringFiles.md [hunk 1]
```diff
@@ -38,8 +38,8 @@ variable names are the same as above, but prefixed with `COMMANDNAME_`.  Example
 env:
  global:
   - MOODLE_BRANCH=MOODLE_32_STABLE
-  - IGNORE_PATHS=vendor/widget,javascript/min-lib.js
```
Comment (kabalin) [inline docs/IgnoringFiles.md:41]: Well spotted, thanks!

File: docs/GHAFileExplained.md [hunk 2]
```diff
@@ -116,6 +116,11 @@ jobs:
       #   IGNORE_PATHS: 'ignore'
       #   IGNORE_NAMES: 'ignore_name.php'
       #   MUSTACHE_IGNORE_NAMES: 'broken.mustache'
```
Comment (kabalin) [inline docs/GHAFileExplained.md:118]: This is a specific command example too, may be remove it.

File: docs/GHAFileExplained.md [hunk 3]
```diff
@@ -116,6 +116,11 @@ jobs:
...
+      #
+      # You also can specify specific command ignores too:
+      #   CODECHECKER_IGNORE_PATHS: 'ignore'
+      #   CODECHECKER_IGNORE_NAMES: 'ignore_name.php'
+      # For more information, see IgnoringFiles.md.
```
Comment (kabalin) [inline docs/GHAFileExplained.md:123]: `For more information` line I would suggest to remove, instead refer to Help page in the line 112. e.g. "For more details on configuring for specific requirements please refer to Help page." Help page contains link to ignore files and other cases such as coverage, plugin dependencies, etc.

**Review by kabalin — CHANGES_REQUESTED** [review]

**Comment by gjb2048** [comment]
@kabalin Thanks for reviewing, I've made changes on https://github.com/moodlehq/moodle-plugin-ci/commit/7e01d0e8c8f74a34e3896e69bc38e2f3ca118530 as a single commit for the pull request.

**Review by kabalin — APPROVED** [review]

**Pull request merged**